webui/package.json

```diff
@@ -9,7 +9,13 @@
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "cypress run",
+    "pree2e": "npm run prepareE2e",
+    "poste2e": "npm run cleanup",
     "cypress": "cypress open",
+    "precypress": "npm run prepareE2e",
+    "postcypress": "npm run cleanup",
+    "prepareE2e": "ts-node cypress/scripts/prepareData.ts",
```

Comment:
nit: casing of `prepareE2e` seems unusual

Reply:
fixed.